pkg/clusters/types/kind/builder.go

```diff
@@ -42,13 +43,23 @@ func (b *Builder) WithClusterVersion(version semver.Version) *Builder {
 	return b
 }
 
+// WithConfig sets a filename containing a KIND config
+func (b *Builder) WithConfig(filename string) *Builder {
```

Comment:
Seems like a good place to drop a helpful link for future code readers:

```suggestion
// WithConfig sets a filename containing a KIND config
// See: https://kind.sigs.k8s.io/docs/user/configuration/
func (b *Builder) WithConfig(filename string) *Builder {
```